css/style.css

```diff
@@ -0,0 +1,442 @@
+
+
+.w-container{
+	max-width: 1300px;
+	margin:auto;
+	background: #fff;
+	margin-bottom: 2rem;
+	padding-bottom: 2rem;
+}
+
+.w-container .navbar-nav{
+	flex:1;
+}
+
+
+.w-container .navbar-nav .nav-item a{
+	font-weight: 700;
+	color:#000;
+	font-size: 1.25rem;
+	padding:0rem 1rem;
+	border-right: 1px solid #e5e5e5;
+}
+
+.logo-area{
+	background: #f72210;
+}
+
+.logo-img{
+	position: relative;
+    top: 10px;
+    transform: scale(1.2);
+}
+
+.current-date{
+	font-weight: 100;
+	color:#fff;
+}
+
+body{
+	background:#F4F5F6;
+	font-family: 'Roboto Condensed', sans-serif;
+}
+
+main{
+	padding:2rem;
+}
+
+.featured-stories-col img{
+	width: 100%;
+}
+
+a{
+	text-decoration:none;
+	color:#161616;
+}
+
+.posts-row{
+	max-width: 98%;
+    margin: auto;
+}
+
+.navbar{
+	border-bottom: 1px solid #e5e5e5;
+}
+
+
+.signin-btn{
+	background: transparent;
+	color:#fff;
+	font-weight: bold;
+}
+
+.subscribe-btn{
+	background: #fff;
+	border-radius: 0;
+	padding:.3rem 1.2rem;
+	font-weight: bold;
+	color:#f72210;
+}
+
+.p-topics{
+	margin-left: .7rem;
+    background: #afe2d2;
+    display: inline-block;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.